### PR TITLE
Create clinic database

### PR DESCRIPTION
**Pull request by ElsonOtake**
- Database clinic created;
- Tables were created based on the supplied diagram;
- Foreign keys were created;
- The `Join table` was created to allow access to many-to-many relationships;
- All the steps are documented on file `schema_based_on_diagram.sql`.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  invoice_id integer REFERENCES invoices(id),
+  treatment_id integer REFERENCES treatments(id),
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE medical_treatments (
+  medical_id integer REFERENCES medical_histories(id),
+  treatment_id integer REFERENCES treatments(id),
+  PRIMARY KEY (medical_id, treatment_id)
+);
```
Comment (Ismailco) [inline schema_based_on_diagram.sql:55]: - [ ] All FOREIGN KEYS should have indexes created, e.g `CREATE INDEX ON medical_histories (patient_id);`, kindly create indexes for all foreign keys

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ElsonOtake, @chumaachike, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (ElsonOtake) [reply schema_based_on_diagram.sql:55]: Thank you for the information. I didn't pay attention to this during my reading.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Chuma & Elson,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Data model follows data diagram

<br>
Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**